src/Sentry/Laravel/EventHandler.php

```diff
@@ -240,13 +240,7 @@ private function addQueryBreadcrumb($query, $bindings, $time, $connectionName)
      */
     protected function logHandler($level, $message, $context)
     {
-        Integration::addBreadcrumb(new Breadcrumb(
-            $level,
-            Breadcrumb::TYPE_USER,
-            'log.' . $level,
-            $message,
-            empty($context) ? [] : ['params' => $context]
-        ));
+        $this->addLogBreadcrumb($level, $message, is_array($context) ? $context : []);
```

Comment:
```suggestion
        $this->addLogBreadcrumb($level, $message, $context);
```

Not better to just leave it like this and let `addLogBreadcrumb` to decide what to do with the input?

Reply:
No, because `addLogBreadcrumb` expects a valid context array. This legacy handler method may receive null as the context though. I would rather prefer that the actual method that adds the breadcrumb can expect valid log data (level, message and context).

Especially, because its legacy.

Reply:
Okay agreed.